fsspec/core.py

```diff
@@ -284,6 +288,11 @@ def open_files(
         protocol=protocol,
         expand=expand,
     )
+    if fs.protocol == "file":
+        fs.auto_mkdir = auto_mkdir
```

Comment:
I do not see an easier way to do this than this fragile line, because the creation of the fs is hidden within the unchaining logic inside the call above.